tests/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "tests"
+version = "0.1.0"
+authors = ["Bryan RHEE <bryan.rhee@hdac.io>"]
+edition = "2018"
+
+[features]
+enable-bonding = ["engine-test-support/enable-bonding"]
+use-as-wasm = ["engine-test-support/use-as-wasm"]
+use-system-contracts = ["engine-test-support/use-system-contracts"]
+
+[dependencies]
+contract = { git="https://github.com/hdac-io/CasperLabs", branch="master", package = "casperlabs-contract", features = ["std"] }
+types = { git="https://github.com/hdac-io/CasperLabs", branch="master", package = "casperlabs-types", features = ["std"] }
+base16 = "0.2.1"
+clap = "2"
+crossbeam-channel = "0.4.0"
+engine-core = { git="https://github.com/hdac-io/CasperLabs", branch="master", package = "casperlabs-engine-core" }
+engine-grpc-server = { git="https://github.com/hdac-io/CasperLabs", branch="master", package = "casperlabs-engine-grpc-server" }
```

Comment:
It seems an unnecessary dependency.

Reply:
https://github.com/hdac-io/swap/pull/1/commits/20729abfb6211eece9df8ae624629722b09c6b6e